tests/transport/rtps_directed_write/.gitignore

```diff
@@ -1,2 +1,6 @@
-/publisher
-/subscriber
+publisher
```

Comment:
I don't think the .gitignore file should be changed by this PR, nothing changing in this PR impacts it.